src/seals/__init__.py

```diff
@@ -28,3 +28,83 @@
         entry_point=f"seals.mujoco:{env_base}Env",
         max_episode_steps=util.get_gym_max_episode_steps(f"{env_base}-v3"),
     )
+
+# Atari
+
+ATARI_ENV_NAMES = [
```

Comment:
Putting this into a constant is a good idea, maybe we can do the same for the mujoco environments for consistency.